apps/oc_chef_wm/itest/setup_helper.erl

```diff
@@ -9,62 +26,23 @@
 
 -export([
          start_server/1,
-         stop_server/1
+         needed_apps/0
         ]).
 
--define(REQUIRED_APPS, [sasl,
-                        asn1,
-                        crypto,
-                        stats_hero,
-                        pooler,
-                        public_key,
-                        ssl,
-                        epgsql,
-                        sqerl,
-                        ibrowse,
-                        ejson,
-                        inets,
-                        mochiweb,
-                        webmachine,
-                        darklaunch,
-                        gen_server2,
-                        eldap,
-                        bcrypt,
-                        folsom,
-                        chef_objects,
-                        rabbit_common,
-                        amqp_client,
-                        gen_bunny,
-                        compiler,
-                        syntax_tools,
-                        goldrush,
-                        lager,
-                        couchbeam,
-                        chef_index,
-                        oc_chef_authz,
-                        oc_chef_wm]).
-
 start_server(Config) ->
```

Comment:
Maybe not in this PR, but we might want to use  the often-ignored app.config that we still ship with oc_erchef - and having reliance on it in the tests will ensure that we keep it accurate.